src/OpenTelemetry/Metrics/BaseExportingMetricReader.cs

```diff
@@ -78,7 +78,7 @@ internal override void SetParentProvider(BaseProvider parentProvider)
         }
 
         /// <inheritdoc/>
-        internal override bool ProcessMetrics(in Batch<Metric> metrics, int timeoutMilliseconds)
+        internal override bool ProcessMetrics(in Batch<Metric> metrics)
```

Comment:
Why do we want to remove it?

Reply:
Just to simplify the method signature because `ProcessMetrics` does not use it.

Reply:
How did you find this?
Is there some stylecop rule that can help us clean up any other similar unused parameters for our private and internal functions?

Reply:
I found this while working on a PR that involved adding logs to Metrics SDK code. There is a code quality rule for this: https://docs.microsoft.com/en-us/dotnet/fundamentals/code-analysis/style-rules/ide0060

However, we don't want to remove this as we would want to modify `BaseExporter.Export` method to add this parameter.